doc/oper.xml

```diff
@@ -1395,6 +1395,38 @@ gap> VerticesReachableFrom(D, 5);
 ]]></Example>
   </Description>
 </ManSection>
+
+<ManSection>
+  <Oper Name="VerticesReachableFrom" Arg="digraph, root"/>
```

Comment:
I think it'd be better if the 2nd argument was called `list`.

Reply:
And the first argument be called `D` for consistency with the other entry you're adding here.